### PR TITLE
Update forms deps

### PR DESCRIPTION
**Pull request by smbea**

### CODE REVIEW

**Comment by smbea** [comment]
@Skaiir please check if the CHANGELOG makes sense from your POV

**Comment by nikku** [comment]
@smbea @marstamm Let's integration test this (as we always do).

**Comment by smbea** [comment]
> @smbea @marstamm Let's integration test this (as we always do).

I already did some integration testing here and didn't find anything.

I also added the missing CHANGELOG entries on top here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ ___Note:__ Yet to be released changes appear here._
 
 ### General
 
+* `FEAT`: add plugin point for bottom panels ([#3998](https://github.com/camunda/camunda-modeler/pull/3998))
+* `FIX`: integrate DRD find ([#3977](https://github.com/camunda/camunda-modeler/pull/3977))
+* `FIX`: use `RewriteFrames` for path rewrite ([#39999](https://github.com/camunda/camunda-modeler/pull/3999))
```
Comment (marstamm) [inline CHANGELOG.md:15]: I don't think we need this in the changelog - it is an internal change that helps us inspect Errors in Sentry, not something user facing 
```suggestion
```

**Comment by marstamm** [comment]
Quick integration test on linux - looks good to me

**Review by marstamm — APPROVED** [review]

**Pull request merged**